htdocs/contact/card.php

```diff
@@ -918,7 +918,7 @@
 				print '<td colspan="3">';
 				$cate_arbo = $form->select_all_categories( Categorie::TYPE_CONTACT, null, null, null, null, 1 );
 				$c = new Categorie( $db );
-				$cats = $c->containing( $object->id, Categorie::TYPE_CONTACT );
+				$cats = $c->containing( $object->id, 'contact' );
```

Comment:
No! Constants are essential to keep separation and allow refactoring without hunting for code like this.